drivers/vhdutil.py

```diff
@@ -28,7 +28,7 @@
 import time
 
 MIN_VHD_SIZE = 2 * 1024 * 1024
-MAX_VHD_SIZE = 2093050 * 1024 * 1024
+MAX_VHD_SIZE = 2088960 * 1024 * 1024
```

Comment:
I wonder whether this would be clearer as

`2040 * 1024 * 1024 * 1024`

or even 

`GB_BYTES = 1024 * 1024 * 1024`
`MAX_VHD_SIZE = 2040 * GB_BYTES`

Reply:
I think  2040*1024*1024*1024 is more in line with the rest of this file, also GB_BYTES (gigabytes_bytes) might be confusing.